app/Services/BaseService.php

```diff
@@ -210,4 +210,13 @@ protected function containerName(): string
 
         return 'TO--' . $this->shortName() . '--' . $this->tag . $portTag;
     }
+
+    public function validateDockerRunTemplate($dockerRunTemplate): string
```

Comment:
I think I'd probably name this something like `sanitize` instead of `validate`

Reply:
Done and will push changes shortly.